### PR TITLE
Update prop.py

### PR DESCRIPTION
**Pull request by pronoym99**
1. Convert ```format``` strings to python3.6+ compliant ```f-strings``` wherever possible.
2. Optimise return statements of functions having rudimentary ```if - else``` statements with equivalent boolean expressions.

### CODE REVIEW

**Comment by pronoym99** [comment]
Please add a hacktoberfest label to this PR if possible.

**Comment by jacadzaca** [comment]
@pronoym99 Could you add yourself to the changelog? Preferably the `minor changes` section

**Comment by pronoym99** [comment]
Yes sure.

**Comment by pronoym99** [comment]
@jacadzaca Done the required changes. Please do have a look.

**Comment by jacadzaca** [comment]
@pronoym99 I did, you've pushed the changes to #481

**Comment by pronoym99** [comment]
@jacadzaca now I have bifurcated [CHANGES](https://github.com/collective/icalendar/blob/master/CHANGES.rst) for each of #481, #482 and #483.

**Comment by pronoym99** [comment]
@jacadzaca bifurcated [CHANGES](https://github.com/collective/icalendar/blob/master/CHANGES.rst) pertaining to each of #481, #482 and #483.

**Comment by jacadzaca** [comment]
@pronoym99 Great, could you now format the changelog as I've asked and put it under the 5.0.2 version, please? Could you also squash your commits, two commits per PR seem resonable -- we don't want to pollute the log with changelog update.

**Comment by pronoym99** [comment]
@jacadzaca I have squashed the commits for this PR and updated the changelog by adding the changes in 5.0.2 as requested by you.

**Comment by jacadzaca** [comment]
@pronoym99 I think your squash didn't work since there are still 7 commits?

**Comment by pronoym99** [comment]
@jacadzaca Yes. I was doing it using the GitKraken app. I need to do so in the cmd line. Maybe it will work there.

**Comment by jacadzaca** [comment]
I'd also phrase the changelog entry differently, e.g `converted all format strings into f-strings in prop.py` or just `refactored prop.py`. The one now is a bit of a mouthful. That's up to you tho :)

File: src/icalendar/prop.py [hunk 1]
```diff
@@ -107,17 +108,12 @@ def dst(self, dt):
...
+
     def utcoffset(self, dt):
-        if self._isdst(dt):
-            return DSTOFFSET
-        else:
-            return STDOFFSET
+        return self._isdst(dt) and DSTOFFSET or STDOFFSET
 
     def dst(self, dt):
-        if self._isdst(dt):
```
Comment (niccokunzmann) [inline src/icalendar/prop.py:117]: If you want to change this:

(A if condition else B) is the way. Not and/or. Logical equivalent but confusing.

Comment (pronoym99) [reply src/icalendar/prop.py:117]: Ok if that is the case. I shall revert back.

**Comment by niccokunzmann** [comment]
Thanks for the PR! I left a comment. Also, this CHANGES file is quite big..

File: src/icalendar/prop.py (hunk 1 above)
Comment (pronoym99) [reply src/icalendar/prop.py:117]: Reverted to ```return A if condition else B``` constructs everywhere.

**Comment by niccokunzmann** [comment]
Thanks for your changes!

**Pull request merged**